drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/scan_card_loader.rb

```diff
@@ -0,0 +1,78 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisExternalApis::TcHmis::Importers::Loaders
+  class ScanCardLoader < BaseLoader
+    def initialize(...)
+      super
+
+      @client_lookup = Hmis::Hud::Client.
+        where(data_source_id: data_source.id).
+        pluck(:personal_id, :id).
+        to_h
```

Comment:
client demographic loader does the same thing, maybe this should also be shared on the base loader